public/app/features/trails/ActionTabs/BreakdownScene.tsx

```diff
@@ -49,25 +53,44 @@ export interface BreakdownSceneState extends SceneObjectState {
   loading?: boolean;
   error?: string;
   blockingMessage?: string;
+  breakdownLayout: BreakdownLayoutType;
 }
 
-export class BreakdownScene extends SceneObjectBase<BreakdownSceneState> {
+export class BreakdownScene extends SceneObjectBase<BreakdownSceneState> implements SceneObjectWithUrlSync {
+  protected _urlSync = new SceneObjectUrlSyncConfig(this, { keys: ['breakdownLayout'] });
   protected _variableDependency = new VariableDependencyConfig(this, {
     variableNames: [VAR_FILTERS],
     onReferencedVariableValueChanged: this.onReferencedVariableValueChanged.bind(this),
   });
 
   constructor(state: Partial<BreakdownSceneState>) {
+    const breakdownLayout = localStorage.getItem(TRAIL_BREAKDOWN_VIEW_KEY);
     super({
       labels: state.labels ?? [],
       ...state,
+      breakdownLayout: isBreakdownLayoutType(breakdownLayout) ? breakdownLayout : 'grid',
     });
 
     this.addActivationHandler(this._onActivate.bind(this));
   }
 
   private _query?: AutoQueryDef;
 
+  getUrlState() {
+    return { breakdownLayout: this.state.breakdownLayout };
+  }
+
+  updateFromUrl(values: SceneObjectUrlValues) {
+    if (typeof values.breakdownLayout === 'string') {
+      const newBreakdownLayout = values.breakdownLayout as BreakdownLayoutType;
+      if (this.state.breakdownLayout !== newBreakdownLayout) {
+        // Change in layout will set up a different set of panel objects that haven't received the current yaxis range
+        this.clearBreakdownPanelAxisValues();
+        this.setState({ breakdownLayout: newBreakdownLayout });
+      }
+    }
+  }
```

Comment:
You should decide where you want this state to be owned/live, BreakdownScene or LayoutSwitcher